library_generation/cli/entry_point.py

```diff
@@ -63,10 +63,21 @@ def main(ctx):
     directory.
     """,
 )
+@click.option(
+    "--api-definition-path",
+    type=str,
+    default=".",
+    show_default=True,
+    help="""
+    The path to which the api definition (proto and service yaml) resides.
+    If not specified, the path is the current working directory.
+    """,
+)
 def generate(
     baseline_generation_config_path: str,
     current_generation_config_path: str,
     repository_path: str,
+    api_definition_path: str,
```

Comment:
I think this is more like a root folder of all api definitions. e.g. The [proto_path](https://github.com/googleapis/sdk-platform-java/blob/main/generation_config.yaml#L40) configure in generation_config.yaml is a relative path to this path right?

Reply:
Maybe change this to `api_definitions_path` or `api_defs_path`?

Reply:
Changed it to `api_definitions_path`.